.github/workflows/prettier.yaml

```
@@ -0,0 +1,26 @@
+name: Apply Prettier style
+
+on:
+  push:
+    paths:
+      - "**.md"
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "20.x"
+      - run: npm install --no-save prettier@3
+      - run: npx prettier --write .
+      - name: git config
+        run: |
+          git config user.name 'github-actions'
+          git config user.email 'github-actions@github.com'
```

https://github.com/orgs/community/discussions/26560#discussioncomment-3531273

```suggestion
          git config user.name "github-actions[bot]"
          git config user.email "41898282+github-actions[bot]@users.noreply.github.com"
```